playbooks/robusta_playbooks/playbook_utils.py

```diff
@@ -0,0 +1,12 @@
+from typing import List
+
+from hikaru.model.rel_1_26 import Pod
+
+
+def pod_row(pod: Pod) -> List[str]:
+    ready_condition = [condition.status for condition in pod.status.conditions if condition.type == "Ready"]
```

Comment:
I think its best if we move the ready condition logic to a static method of RobustaPod class.

Reply:
Moved the logic out to the RobustaPod class